addons/l10n_es_edi_tbai/tests/test_edi_xml.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+
+from freezegun import freeze_time
+from lxml import etree
+from odoo.addons.l10n_es_edi_tbai.models.xml_utils import L10nEsTbaiXmlUtils
+from odoo.tests import tagged
+
+from .common import TestEsEdiTbaiCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestEdiTbaiXmls(TestEsEdiTbaiCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.out_invoice = cls.env['account.move'].create({
+            'name': 'INV/01',
+            'move_type': 'out_invoice',
+            'invoice_date': datetime.now(),
+            'partner_id': cls.partner_a.id,
+            'invoice_line_ids': [(0, 0, {
+                'product_id': cls.product_a.id,
+                'price_unit': 1000.0,
+                'quantity': 5,
+                'discount': 20.0,
+                'tax_ids': [(6, 0, cls._get_tax_by_xml_id('s_iva21b').ids)],
+            })],
+        })
+
+        cls.edi_format = cls.env.ref('l10n_es_edi_tbai.edi_es_tbai')
```

Comment:
@jco-odoo Not useful since it is set by AccountEdiTestCommon from the edi_format_ref param ?

Reply:
Just check if it is used somewhere maybe...